webrender/src/texture_cache.rs

```diff
@@ -581,6 +595,17 @@ impl TextureCache {
          DeviceUintRect::new(origin, entry.size))
     }
 
+    pub fn mark_unused(&mut self, handle: &TextureCacheHandle) {
+        if let Some(ref handle) = handle.entry {
+            if let Some(entry) = self.entries.get_opt_mut(handle) {
+                // Set a very low last accessed frame to make it very likely that this entry
+                // will get cleaned up next time we try to expire entries.
+                entry.last_access = FrameId(0);
+                entry.eviction = Eviction::Auto;
```

Comment:
given that `last_access` doesn't make sense for `Manual` anyway, can we make it a field of `Eviction::Auto`?

Reply:
The `Eviction` enum was primarily meant to avoid adding a boolean parameter to the update function so I'd rather keep it this way.

Reply:
I don't understand. If we can get a better semantics, better use of types, then why neglect it?

Reply:
If you insist I can add another enum for the thing stored in here that has a last access time and keep this enum for the signature fo `TextureCache::update`. Where `Eviction` shows up most right now is as an argument in the various calls to `TextureCache::update` where the last access time member doesn't make sense (I think).

Reply:
ok, fair enough. I can follow up with this myself if I see it feasible